재하_박준하/0주차_미션/css/미션2/style.css

```
@@ -0,0 +1,124 @@
+.wrapper {
```

```suggestion
body {
    margin: 0;
    padding: 0;
}
```

이걸 추가하면 좀 더 예쁘게 구현될 것 같습니다!

---

감사합니다!
기존 코드를 개발자 도구에서 확인했을 때 body에서 margin 값이 8씩 있어서 margin이 없을 때 화면에 더 꽉 차도록 랜더링 되는점은 확인되었는데, padding property는 어떤 이유 때문에 추가하셨는지 알 수 있을까요!?

---

각 브라우저마다 HTML 요소에 기본적으로 제공하는 스타일이 있어서 그러한 스타일에 영향을 받지 않고 개발하기 위해서 대부분 프로젝트를 할 때 css 스타일을 초기화 하고 진행합니다! 사실 해당 예제에서는 padding: 0은 설정하지 않아도 예쁘게 잘 나오지만, 이렇게 적용이 되지 않는 경우가 간혹 있습니다! 'css 스타일 초기화'에 대해서 알아보시면 좋을 것 같아요!

---

css 미션2 수정 및 js 미션 1 push 했습니다!